api/app/utils/fba_calculator.py

```diff
@@ -133,6 +138,9 @@ def calculate_fire_behaviour_advisory(station: FBACalculatorWeatherStation) -> F
         # The day 144 is the average date for the minimum foliar moisture content in the boreal regions of
         # Canada. It is usually pretty close maybe 7 days in either direction.
         date_of_minimum_foliar_moisture_content = 144
+    elif station.fuel_type == 'C3':
+        # TODO: Confirm if this is valid
```

Comment:
does this TODO still apply? I thought you had already validated it?

Reply:
No - didn't get an answer from Dana, and haven't reached out to DP.

Reply:
:) completely removed. happy times!